### PR TITLE
Add messageId

### PR DESCRIPTION
**Pull request by TordJoranger**

### CODE REVIEW

File: src/Equinor.ProCoSys.Completion.Domain/Events/IntegrationEvents/HistoryEvents/HistoryDeletedIntegrationEvent.cs [hunk 1]
```diff
@@ -9,4 +10,7 @@ public record HistoryDeletedIntegrationEvent(
     Guid Guid,
     Guid? ParentGuid,
     User EventBy,
-    DateTime EventAtUtc) : IHistoryItemDeletedV1;
+    DateTime EventAtUtc) : IHistoryItemDeletedV1
+{
+    public Guid MessageId { get; }  = NewId.NextGuid();
```
Comment (janid1967) [inline src/Equinor.ProCoSys.Completion.Domain/Events/IntegrationEvents/HistoryEvents/HistoryDeletedIntegrationEvent.cs:15]: expr.body

File: src/Equinor.ProCoSys.Completion.Domain/Events/IntegrationEvents/HistoryEvents/HistoryUpdatedIntegrationEvent.cs [hunk 2]
```diff
@@ -11,4 +12,7 @@ public record HistoryUpdatedIntegrationEvent(
     Guid? ParentGuid,
     User EventBy,
     DateTime EventAtUtc,
-    List<IChangedProperty> ChangedProperties) : IHistoryItemUpdatedV1;
+    List<IChangedProperty> ChangedProperties) : IHistoryItemUpdatedV1
+{
+    public Guid MessageId { get; }  = NewId.NextGuid();
```
Comment (janid1967) [inline src/Equinor.ProCoSys.Completion.Domain/Events/IntegrationEvents/HistoryEvents/HistoryUpdatedIntegrationEvent.cs:17]: e.body

File: src/Equinor.ProCoSys.Completion.MessageContracts/History/IHistoryItemCreatedV1.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Generic;
+﻿using System;
```
Comment (janid1967) [inline src/Equinor.ProCoSys.Completion.MessageContracts/History/IHistoryItemCreatedV1.cs:1]: necessary?

**Review by janid1967 — APPROVED** [review]
Approved, after fixing red tests

**Review by DownGoat — APPROVED** [review]

**Pull request merged**